ArduPlane/ArduPlane.cpp

```diff
@@ -690,4 +690,14 @@ bool Plane::get_target_location(Location& target_loc)
     return false;
 }
 
+// correct AHRS pitch for TRIM_PITCH_CD in non-VTOL modes
+float Plane::get_osd_pitch_rad() const
+{
+   float pitch = ahrs.pitch;
+   if (!quadplane.in_vtol_mode()) {
+      pitch += g.pitch_trim_cd * 0.01 * DEG_TO_RAD;
```

Comment:
Okay...did not know that constant existed...

Reply:
Try below. The base SITL model has -2.74deg pitch in steady flight. So if we add -2.74 + -2.74 we wont't get what we want. Unless the trim_convention is opposite to this? But when playing with sitl using mission planner &osd. Mission planner appears to use trim_pitch as used below?

```suggestion
      pitch -= g.pitch_trim_cd * 0.01 * DEGX100;
```

Reply:
The reason for this change from your previous code is because previously you had the code below this,
```cpp
    float pitch = -ahrs.pitch;
```
So choose which sign to change? I would suggest what I have above as it is easier to read. 

https://github.com/Hwurzburg/ardupilot/blob/186fc62ecada81deb74f2c99196c33ddb1e443ca/libraries/AP_OSD/AP_OSD_Screen.cpp#L1462

Reply:
Yeah quite possibly could have screwed up a sign I'll check it when I get back home. Yeah having the wrong sign would just double the effect that I'm trying to correct. Thanks

Reply:
changing a sign fixed it...thanks